torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1339,6 +1341,71 @@ def call_function(
         )
 
 
+class StrictModeHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ) -> "VariableTracker":
+        from .builder import wrap_fx_proxy
+        callable = args[0]
+        graph_checkpoint, checkpoint = tx.output.graph, tx.copy_graphstate()
```

Comment:
Using copy_graphstate is not recommended any more [here](https://github.com/pytorch/pytorch/issues/113235). We could just delete this line and keep most of the mutations and restoring the side-effects would be good.

Reply:
ok